lib/RESTClient.js

```diff
@@ -336,6 +289,7 @@ class RESTClient {
      * @params {function} callback - callback
      */
     request(method, beginPath, log, params, data, callback) {
+        log.debug('sending request', { httpMethod: method, beginPath });
         let path = beginPath;
         assert(typeof callback === 'function', 'callback must be a function');
         const headers = {
```

Comment:
Not related to this PR but shouldn't `content-length` be equal to `data.length` in the case `data` is available?